executor/setup.py

```diff
@@ -27,5 +27,8 @@
         'console_scripts': [
             'cook-executor = cook.__main__:main'
         ]
-    }
+    },
+    classifiers=[
+        "Programming Language :: Python :: 3"
```

Comment:
Should we be more specific?
```suggestion
        "Programming Language :: Python :: 3.5"
```